submissions/madmaxWMFU/google-popup/index.html

```diff
@@ -10,7 +10,7 @@
 
 <body>
     <header>
-        <div class="mainContainer">
+        <div class="navigationContainer">
             <div class="simpleNav">
                 <div class="divLinks">
                     <div><a class="link mail" href="#">Gmail</a></div>
```

Comment:
Avoid using `<div>` without class as wrapper/container especially if you add style to him.
Also, in CSS you have to make a more specific selector.

Reply:
Done!